test/fishjam_web/controllers/component_controller_test.exs

```diff
@@ -73,11 +75,11 @@ defmodule FishjamWeb.ComponentControllerTest do
     end
 
     test "deletes component from not exisiting room", %{conn: conn, component_id: component_id} do
-      room_id = "abc"
+      room_id = ID.generate()
```

Comment:
Why do we need to make this change?

Reply:
Without it, tests like this one failed on the `decode_node_name` part in `room/id.ex` (when running with `REQUEST_ROUTING=true`)

Reply:
Ok, so from this PR, we remove the possibility of passing custom room ID?

Reply:
No, why? The possibility is still there, this just had to do with the fact that this test could be run both with and without request routing, and so I fixed it to pass in both cases